infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -318,8 +339,21 @@ func (b *TFBlueprintTest) DefaultInit(assert *assert.Assertions) {
 	}))
 }
 
+// DefaultTerraformVetVet runs TF plan, TF show, and gcloud terraform vet on a blueprint.
+func (b *TFBlueprintTest) TerraformVet(assert *assert.Assertions) {
+	terraform.Plan(b.t, b.GetTFOptions())
+	jsonPlan := terraform.Show(b.t, b.GetTFOptions())
+	filepath, err := utils.WriteTmpFileWithExtension(jsonPlan, "json")
+	require.NoError(b.t, err)
```

Comment:
Lets use `assert` otherwise the test will fail fast.

Reply:
fixed

Reply:
@daniel-cit looks like this is still require?

Reply:
I had replaced the one after the call to `gcloud.TFVet` and I forgot this one. 
Fixed now.